module-sys/SystemManager/SystemManagerCommon.cpp

```diff
@@ -612,7 +612,7 @@ namespace sys
             auto msg = static_cast<sys::SentinelRemovalMessage *>(message);
             powerManager->RemoveSentinel(msg->getSentinelName());
 
-            return sys::MessageNone{};
+            return std::make_shared<sys::ResponseMessage>();
```

Comment:
This must-have hurt :(

Reply:
...and wondering why a sync message doesn't work... :D